base_product_mass_addition/models/product_product.py

```diff
@@ -13,25 +13,53 @@ class ProductProduct(models.Model):
 
     qty_to_process = fields.Float(
         compute="_compute_process_qty",
-        inverse="_inverse_set_process_qty",
+        inverse="_inverse_quick_vals",
```

Comment:
@kevinkhao Is that change wanted as module is already merged ?

Reply:
thanks for feedback
reverted https://github.com/OCA/product-attribute/pull/864/commits/f9c49b8d193d248cfdf81e17f938cca793b09142